Lombiq.UIKit/Lombiq.UIKit.csproj

```
@@ -35,7 +35,10 @@
     <PackageReference Include="OrchardCore.Contents" Version="2.1.0" />
     <PackageReference Include="OrchardCore.DisplayManagement" Version="2.1.0" />
     <PackageReference Include="OrchardCore.ResourceManagement" Version="2.1.0" />
-    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="3.0.71" />
+    <!-- This is commented out, because restoring Slick via LibMan is flaky, see
+    https://github.com/Lombiq/GitHub-Actions/issues/549. So, we keep the library in the repository. This should only be
+    temporarily uncommented while updating the Slick snapshot. -->
+    <!--<PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="3.0.71" />-->
```

Instead of commenting out, why not just deactivate in GH specifically?
```suggestion
    <PackageReference Include="Microsoft.Web.LibraryManager.Build" Version="3.0.71" Condition="'$(GITHUB_ENV)' == ''" />
```

This is a nicer dev experience, it will prompt Renovate to keep the package up to date, and you won't need the "This should only be temporarily uncommented while updating the Slick snapshot." comment nor the `note2` in the JSON file.

---

Ah, that's clever.

---

Actually, the files shouldn't be touched locally either. We don't gain anything by it except for the rare cases of updates, but since LibMan touches the files, they'll show up as changed in Git (at least in Git Extensions), even though their contents didn't change.

---

That sounds like a bad CRLF config on your end, but okay.

---

Perhaps, but it's then the fault of some tooling, because we have the _.gitattributes_ config for this.